blog/BCHN-lead-maintainer-report-2020-06-15.md

```diff
@@ -0,0 +1,350 @@
+
+This report will replace this "Financial Overview" section going forward (I will only reference it in future reports).
+
+Our full Financial Report contains links to all our previous financial overviews and allows us to go into more financial details than would be suitable for this maintainer report.
+
+BCHN intends to publish these Financial Reports on a monthly basis, allowing anyone to see how the money donated to us is being spent.
+
+We do not shy away from transparency and accountability. No other Bitcoin Cash full node software project I know of releases such detailed financial information. I am confident in claiming that BCHN is leading by example here.
+
+## [Project work items update](#project-work-items-update)
+Since initial release:
+*   Issues: 123 raised, 57 closed, 66 open
+    
+*   Merge Requests: 545 raised, 455 merged, 28 closed (read: rejected or aborted), 62 open
+
+Since last maintainer report (2020-04-15):
+*   Issues: 47 raised, 18 closed, 29 open
+    
+*   Merge Requests: 394 raised, 325 merged, 13 closed (read: rejected or aborted), 56 open
+
+### [Backporting](#backporting)
+What probably stands out most from the above numbers is a sharp increase in Merge Requests over this period. This is due to a concerted effort (still underway) to tackle outstanding backports from upstream projects.
+
+Backporting isn't simply copying changes from upstream, it requires experienced developers to review and test (and in some instances: develop tests for) those changes. In some instances we have found problems and reject upstream changes.
+
+We are currently still playing catch up. Both Core and ABC have more developers, so we need to figure out how to work smart.
+
+The graph below gives an overview of our backporting activities.
+
+<figure class="text-center">
+    <img src="/static/img/newsroom/2020-06-15/lR0FGkOAhTyIFTxBaH4JWgtTww5EyLee2YaYD4eU.png" class="rounded img-fluid" alt="" title="">
+</figure>
+
+We need to ramp those up further (to make dents in yellow and red lines and bump up the green line).
+
+The slope of the increasing red line indicates rate at which backports from upstream (in this case, ABC) are heading towards us. They then get sorted into various categories (planned, deferred, rejected) and planned ones get worked on until they are integrated (merged).
+
+Internal project discussions were in May and held and concluded that the backporting effort would, for now, be shared by existing contributors to the project. This may change at a later date, but two BCHN developers have firmly committed time to this activity, while another developer has flexibly contributed to it. This is in accordance with our Flipstarter proposal, where we did foresee that
+
+> we may either share it across developers or contract further development resources with these funds.

```

Comment:
Another quoted paragraph which is badly formatted in the rendered HTML.

Please take similar action as to you will with the previous such instance.